_src/Evasions/techniques/AVBypass.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Evasions: Bypassing Security Solutions"
+title-image: "/assets/icons/AVBypass.svg"
+categories: evasions
+tags: AVBypass
+---
+
+<h1>Contents</h1>
+
+[Bypassing Security Solutions detection methods](#avbypass-detection-methods)
+<br />
+[1. Detect Wine](#detect-wine)
+<br />
+[2. Add to Windows defender exclusion list](#add-to-windows-defender-exclusion-list)
+<br />
+[3. Patch EtwEventWrite](#patch-etweventwrite)
```

Comment:
Do you think this can be moved into Hooks section? Right now there is only a description of checking hooks, but function modification can be added as well